using-the-website/bonds.md

```diff
@@ -1,30 +1,61 @@
 # Bonds
 
-Bonds allow users to buy OHM from the protocol for a discount by trading it for liquidity \(SLP Tokens\). Bonds take roughly 15 epochs to vest; during that period, the OHM the user is due is slowly released to them. Bonds help the protocol accumulate and lock liquidity.
+Bonds allow users to buy OHM from the protocol at a discount by trading it with i) liquidity \(LP tokens\) or ii) DAI. Bonds take roughly 15 epochs to vest, and OHM tokens are vested linearly to the user over that period. LP bond helps the protocol accumulate and lock liquidity, while DAI bond allows the protocol to grow its treasury faster.
 
 ## How to Bond
 
-First, go to Sushiswap and add liquidity. If you do not know how to do so, there is a good guide here. Next, come to our [website](http://olympusdao.finance/#/bond). Make sure "Bond" is selected. Type in the amount of SLP you would like to bond, then click Approve and sign the transaction. Next, click Bond and sign that transaction. **Please note that bonding takes two transactions; approve and bond.**
+#### LP Bond
+
+1. Go to Sushiswap and add liquidity. If you do not know how to do so, there is a good guide on [SushiSwap documentation](https://help.sushidocs.com/guides/how-to-add-tokens-to-sushiswap-exchange-as-an-lp).
+
+2. Head to [Olympus website](http://olympusdao.finance/#/bond). There are two tabs: "Bond" and "Redeem"; make sure "Bond" is selected.
+
+3. Type in the amount of SLP you would like to bond, then click "Approve" and sign the transaction.
+
+4. After the "Approve" transaction has been processed successfully, click "Bond" and sign the transaction. Voila, you have purchased your first SLP bond!
+
+#### DAI Bond
+
+1. Head to [Olympus website](http://olympusdao.finance/#/bondai). There are two tabs: "Bond" and "Redeem"; make sure "Bond" is selected.
+
+2. Type in the amount of DAI you would like to bond, then click "Approve" and sign the transaction.
+
+3. After the "Approve" transaction has been processed successfully, click "Bond" and sign the transaction. Voila, you have purchased your first DAI bond!
+
+*Note: The "Approve" transaction is only needed when bonding for the first time; subsequent bonding only requires you to perform the "Bond" transaction.*
 
 ## **How to Redeem**
 
-Come to our [website](http://olympusdao.finance/#/bond) and select the "Redeem" tab at the top of the window. Click "Redeem" to claim all of your available rewards.
+On the same webpage that you used to purchase bond, select the "Redeem" tab. Click "Redeem" to claim all of your available rewards.
```

Comment:
I would find it more useful to keep the link to the page. Referring to another section of the docs now may mean more cognitive load, especially if somebody is just looking for this section. Imagine you did not read the docs above but just want to get going with redemption.

Reply:
Agree, I would include the links.